micro-rdk/src/common/robot.rs

```diff
@@ -265,6 +274,11 @@ impl LocalRobot {
             data_manager_collection_task: None,
             start_time: Instant::now(),
         };
+        if let Some(cloud_cfg) = config.cloud.as_ref() {
```

Comment:
Under what circumstances will `config.cloud()` actually be `None`? Should it be possible to build a `LocalRobot` from a `config` lacking a `cloud` data?

Reply:
I think you should be able to from a library perspective, we can create a robot without starting a server. But maybe that means this shouldn't be stored on the robot?